template/metadata.yaml.tpl

```diff
@@ -0,0 +1,8 @@
+service:
+  full_name: "{{ .ServiceFullName }}"
+  short_name: "{{ .ServiceID }}"
```

Comment:
I would prefer if we don't generate these values - or at least we should include instructions for the service teams to check and update them. The purpose of this file was that the developers had final say over the displayed names of their controllers. ServiceID is not always the service abbreviation, and is not always capitalised correctly.